# Announces a task to the robots.

uint32 request_id
uint32[] start_ids
uint32[] end_ids
time timeout
